x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -408,7 +408,13 @@ static RoleDescriptor kibanaSystem(String name) {
                 getRemoteIndicesReadPrivileges("traces-apm-*") },
             null,
             null,
-            null
+            "Grants access necessary for the Kibana system user to read from and write to the Kibana indices, "
+                + "manage index templates and tokens, and check the availability of the Elasticsearch cluster. "
+                + "It also permits activating, searching, and retrieving user profiles, "
+                + "as well as updating user profile data for the kibana-* namespace. "
+                + "Additionally, this role grants read access to the .monitoring-* indices "
+                + "and read and write access to the .reporting-* indices. "
+                + "Note: This role should not be assigned to users as the granted permissions may change between releases."
```

Comment:
This is a verbatim copy of [`kibana_system` role's description](https://www.elastic.co/guide/en/elasticsearch/reference/current/built-in-roles.html#:~:text=tools%20in%20Kibana.-,kibana_system,-Grants%20access%20necessary). The documentation seems to be outdated, since the permissions have changed quite a lot since the documentation was first written. 
Not sure if we should cover them all. But I do think it's worth revisiting description to be more explicit in stating that this its intention is for system use by Kibana  and that it should not be granted to regular users.

Reply:
Ah good catch! I can take this back to my team to discuss updates, but I think this is fine for now.

Reply:
Sounds good to me.